mavros/src/plugins/sys_status.cpp

```diff
@@ -738,6 +743,55 @@ class SystemStatusPlugin : public plugin::Plugin
     }
   }
 
+  /**
+   * Sent EVENT message to rosout
+   *
+   * @param[in] severity
+   */
+  void process_event_normal(uint8_t severity, uint32_t px4_id, const std::array<uint8_t, 40>& arguments)
```

Comment:
As i remember message class has type definition for arrays, better to use it.

Reply:
From what I see in `mavlink_msg_event.hpp`, they use `std::array<uint8_t, 40> arguments;` the same way. Anything I should change here?

Reply:
That's okay.